<?xml version="1.0"?>
<!--
  Launch wit driver
-->
<launch>
  
  <!-- defined default address and refreshrate -->
  <arg name="refresh_rate"  default="10"/>
  <arg name="port" default="/dev/ttyUSB0"/>
  <arg name="baudrate"  default="460800"/>


<!-- run node -->
  <node name="wit_driver1" pkg="wit_driver" type="wit_driver" output="screen">
    <param name="address" type="int" value="80" />
    <param name="refresh_rate" type="int" value="$(arg refresh_rate)" />
    <param name="modbus_service" type="str" value="modbus_service1" />
  </node>

  <node name="modbus_master_serial" pkg="wit_driver" type="modbus_master_serial" output="screen">
    <param name="port" type="str" value="$(arg port)" />
    <param name="modbus_service" type="str" value="modbus_service1" />
    <param name="baudrate" type="int" value="$(arg baudrate)" />
  </node>
</launch>
